udata/tests/features/territories/test_territories_process.py

```diff
@@ -367,3 +382,24 @@ def test_with_region_datasets(self):
         self.assertFalse(self.get_context_variable('has_pertinent_datasets'))
         self.assertEqual(self.get_context_variable('other_datasets'), [])
         self.assertNotIn('dataset-item--cta', data)
+
+    def test_with_old_region_datasets(self):
+        lr, occitanie = create_old_new_regions_fixtures()
+        with self.autoindex():
+            for region in [lr, occitanie]:
+                organization = OrganizationFactory(zone=region.id)
+                for _ in range(3):
```

Comment:
Sorry, it's `VisibleDatasetFactory.create_batch(3, **kwargs)`, build_batch does not save